shell/pages/home.vue

```diff
@@ -74,6 +84,10 @@ export default {
       return this.$store.getters['management/all'](CAPI.RANCHER_CLUSTER);
     },
 
+    hasPodSecurityPolicies() {
+      return this.psps && this.psps.length;
```

Comment:
Also want to only show the banner if the cluster is k8s version 1.21 or greater.